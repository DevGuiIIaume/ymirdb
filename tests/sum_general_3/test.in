set a 1
set b 2
sum a
sum b
push a b
sum a
set c 3
push b c
sum b
append b 5
sum b
list entries
bye
