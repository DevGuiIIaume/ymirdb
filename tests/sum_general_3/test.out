> ok

> ok

> 1

> 2

> ok

> 3

> ok

> ok

> 5

> ok

> 10

> c [3]
b [c 2 5]
a [b 1]

> bye
